lib/bot/cmds/thanks.js

```diff
@@ -106,7 +106,7 @@ const thanksCommands = {
       const username = blob.response.about.username,
             about = blob.response.about,
             uri = 'http://www.freecodecamp.com/' + username;
-      str = '> :star: ' + about.browniePoints + ' | @' + username + ' | ';
+      str = '> :cookie: ' + about.browniePoints + ' | @' + username + ' | ';
```

Comment:
@noncentz can you implement this https://github.com/FreeCodeCamp/camperbot/issues/86 here? (_optional_)